internal/deviations/deviations.go

```diff
@@ -625,3 +625,10 @@ func ISISLspMetadataLeafsUnsupported(dut *ondatra.DUTDevice) bool {
 func QOSQueueRequiresID(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetQosQueueRequiresId()
 }
+
+// GetEcnSameMinMaxThresholdUnsupported returns true for devices that don't support Minimum and Maximum threshold values are same
+// EcnMaxMimumThreshold returns the maximum threshold value while comparing for pass or fail conditions.
```

Comment:
I didn't see this function "EcnMaxMimumThreshold" defined.

Reply:
corrected